src/dmd/parse.d

```diff
@@ -2640,7 +2634,7 @@ final class Parser(AST) : Lexer
             check(TOKrparen);
         }
 
-        auto fbody = parseStatement(PScurly);
+        auto fbody = parseStatement(ParseStatementFlags.curly);
         auto f = new AST.InvariantDeclaration(loc, token.loc, stc, null, fbody);
```

Comment:
`ParseStatementFlags` is quite long. How about adding a private `alias` at the top of the module:
```D
private alias PS = ParseStatementFlags;
//...
auto fbody = parseStatement(PS.curly);
```

Reply:
I disagree, it's not used *that* much.

Reply:
I'm OK with or without the alias.